calSoma.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class Programa_soma {
+
+   
+    public static void main(String[] args) 
+    {
+       int x = 0 , y = 0, soma;
```

Comment:
```suggestion
       int x = 0 , y = 0, soma;
```
podemos acrescentar mais algumas variáveis para utilizar na calculadora soma.